backend/web/static/lang/activities_fr.js

```diff
@@ -561,8 +561,8 @@ desc:      "Concentrez-vous sur les aspects positifs au lieu des problèmes en d
 <ul>\
   <li>Trouvons plusieurs façons de renforcer notre travail d'équipe et nos process</li>\
   <li>Trouvons comment étendre nos bonnes pratiques et méthodes d'ingénierie</li>\
-  <li>Nous identifierons et tenterons de trouver plus de rapports dans le travail qui fonctionnent</li>\
-  <li>Nous identifierons là où nous avons produit le plus de valeur ajoutée au cours de la dernière itération afin d'agumenter celle que nous fournirons lors du prochain</li>\
+  <li>Identifions et tentons de trouver plus de rapports dans le travail qui fonctionne</li>\
+  <li>Identifions là où nous avons produit le plus de valeur ajoutée au cours de la dernière itération</li>\
```

Comment:
La traduction de la fin de la phrase a été supprimée.
End of the translation has been truncated, it should be reinjected.

Reply:
Chunk to readd " afin d'augmenter la valeur fournie lors de la prochaine itération"

Reply:
Oui, je trouve la phrase mieux sans cette partie (qui n'apporte rien), mais je suis d'accord que ça sort du cadre d'une simple relecture typo. Je remets.

Reply:
Done!